server/graphql/v2/object/ConnectedAccount.ts

```diff
@@ -31,5 +34,47 @@ export const GraphQLConnectedAccount = new GraphQLObjectType({
     },
     settings: { type: GraphQLJSON },
     service: { type: new GraphQLNonNull(GraphQLConnectedAccountService) },
+    accountsMirrored: {
+      type: new GraphQLList(GraphQLAccount),
```

Comment:
Doesn't matter much for now, but that will become an issue when we'll enable TS strict mode

```suggestion
      type: new GraphQLNonNull(new GraphQLList(GraphQLAccount)),
```